monte_carlo.ipynb

```diff
@@ -174,7 +174,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 35,
```

Comment:
Please commit a cleared or ran-once notebook and I’ll merge.

Reply:
Done